android-guide.md

```diff
@@ -36,20 +36,33 @@ dependencies {
     classpath 'com.android.tools.build:gradle:2.1.2' // <--- update this
     classpath 'com.google.gms:google-services:3.0.0' // <--- add this
 }
+...
+allprojects {
+    repositories {
+        mavenLocal()
+        maven {url "https://maven.google.com"} // <--- this should be added and should be above jcenter
+        jcenter()
+        google()
+        maven {
+            // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
+            url "$rootDir/../node_modules/react-native/android"
+        }
+    }
+}
 ```
 
 * Update `android/app/build.gradle` with
 
 ```gradle
 ...
 dependencies {
-    compile fileTree(dir: "libs", include: ["*.jar"])
-    compile "com.android.support:appcompat-v7:23.0.1"
-    compile "com.facebook.react:react-native:+"
-    compile(project(":react-native-google-signin")){         
+    implementation fileTree(dir: "libs", include: ["*.jar"])
```

Comment:
That's right - I was coming from the perspective of doing a fresh install of Android Studio. Should there maybe be a gradle > 3 section since several implementation details are different?

Reply:
Personally I'd prefer to keep things simple and have just one setup section that describes usage with gradle 3. So just updating the `com.android.tools.build:gradle` part should be good I think.